nodes/parachain/src/command.rs

```diff
@@ -346,14 +373,16 @@ pub fn run() -> Result<()> {
 				);
 
 				match cli.runtime.as_str() {
-					"peregrine" => crate::service::start_node::<MashRuntimeExecutor, peregrine_runtime::RuntimeApi>(
-						config,
-						polkadot_config,
-						id,
-					)
-					.await
-					.map(|r| r.0)
-					.map_err(Into::into),
+					"peregrine" => {
```

Comment:
could we use chain identity here instead?

Reply:
Sure, good catch! [a5e9d01](https://github.com/KILTprotocol/mashnet-node/pull/319/commits/a5e9d01128c1c5eb2c0761278f41fe1ac282911d)